shared/output/cloud/language_v1/test/google/cloud/language/v1/language_service_test.rb

```diff
@@ -85,19 +85,24 @@ def test_analyze_sentiment
       end
 
       # Use protobuf object
-      client.analyze_sentiment ::Google::Cloud::Language::V1::AnalyzeSentimentRequest.new(document: document, encoding_type: encoding_type) do |response, operation|
+      client.analyze_sentiment ::Google::Cloud::Language::V1::AnalyzeSentimentRequest.new(document: document,
```

Comment:
should this really stick back to the beginning of the line like this?

Reply:
( the comment above is about the next line, where the `encoding_type: encoding_type` got moved on line break )

Reply:
Fixed. Looks like we failed to disable `Layout/LineLength` when doing formatting fixup. (And Rubocop's "fixing" of line length apparently sucks.)